cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -133,6 +133,38 @@ rules:
   - "*"
   verbs:
   - "*"
+- apiGroups:
```

Comment:
We need to make sure these permissions are added to the upgrade as well. I believe these should be added to `upgrade-from-v1.0-apply.yaml`, although I don't see that file. @BlaineEXE Isn't that the expected file for updated RBAC? Or where should it go?

Reply:
New resources that don't exist for v1.0 clusters should go into `upgrade-from-v1.0-create.yaml`

Reply:
Would this be a good time to create another set of rules that aggregate to this instead of modifying an existing ruleset?

Reply:
Have we found a use for the aggregation rules during upgrade? If we can get by with the apply (or some other method during upgrade), @davidvossel mentioned that OLM generation would be much simpler without the aggregation rules.

Reply:
right, it's unclear to me what the aggregation rules are trying to solve during an upgrade.  

aggregation makes sense when we need to add permissions to a common role binding that our component doesn't directly controll... So for example adding the ability for everyone with the default "edit" role to gain a new permission. We'd do this using aggregation instead of modifying the "edit" role directly. 

When we control the creation of the ServiceAccount, [Cluster]Role and [Cluster]RoleBinding, I'm unaware of any benefit that aggregation provides us.